polaris.shopify.com/content/patterns/new-features/variants/default.md

```diff
@@ -0,0 +1,46 @@
+---
+hideFromNav: true
+---
+
+## When to highlight new features
+
+Features should have all 3 of the following characteristics before highlighting them to merchants with a New Badge or PIP:
+
+- Shopify wants to drive adoption of this feature because it has high business value
+- It’s creating new and outsized value for merchants (not just an improved way of doing something they are already doing)
+- It’s worth distracting merchants from their current workflow to inform them about a whole new part of Shopify
+
+Things that generally meet this criteria are large and impactful. Examples could be entire new pages in settings or the introduction of an entirely new technology or capability. 
+
+## Avoid highlighting individual features within a page
+
+As the Shopify admin UI continues to evolve, it's natural for teams to want to draw attention to new features and improvements. However, we strongly advise against using "new" badges or blue pips to highlight these additions within individual sections like Products, Orders, and Customers.
+
+
+### Why this should be avoided
+
+  **Inconsistency** - Different teams might use different methods to highlight features, leading to an inconsistent experience for merchants.
+  
+  **Visual clutter** - Over time, the admin UI may become cluttered with numerous badges and pips, making it difficult for merchants to focus on their primary tasks.
+
```

Comment:
Formatting: This should probably just be h4s (####)

Reply:
Crikey, that's awful but I'd still ship the h4s. Avoiding the h4s only hides the issue and prevents it from being fixed.